p4proto/kctrl/switchapi/switch_pd_utils.h

```diff
@@ -57,10 +57,22 @@ bf_status_t switch_pd_deallocate_handle_session(bf_rt_table_key_hdl *key_hdl_t,
 
```

Comment:
If these are not being used, we can remove the declarations

Reply:
Removed